bigbluebutton-html5/private/config/public/app.yaml

```diff
@@ -53,4 +53,4 @@ app:
 
   # The initial client version has limited moderator capabilities
   # The following flag disables moderator-only features
-  allowHTML5Moderator: false
+  allowHTML5Moderator: true
```

Comment:
:thinking:

Reply:
I think we *should* set the flag to true

Reply:
This way we'll be able to set a user to be presenter (for a html5 complete meeting)